led_interface.py

```diff
@@ -1,21 +1,22 @@
 import collections
+from chess_position import ChessPosition
 
 class LedInterface():
 
     def __init__(self):
-        pass
+        self.chess_pos = ChessPosition(self, col, row)
```

Comment:
Making the class in your `__init__()` method should only be used for classes which represent systems you will need to control. Such as the LCD screen, or LEDs. 

The `ChessPosition` and `ChessMove` classes we made represent a position on the Chess board. So you  should use them whenever you need to represent a move. So for your class that would be in the `turn_on_led`, `turn_off_led`, `start_blinking_led`, and `stop_blinking_led` methods.

Delete this line.